src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.spec.ts

```diff
@@ -126,6 +132,50 @@ describe('PaypalCommerceButtonStrategy', () => {
         expect(paypal.Buttons).toHaveBeenCalledWith({
             createOrder: expect.any(Function),
             onApprove: expect.any(Function),
+            style: paypalOptions.style,
+        });
+    });
+
+    it('validate style for PaypalCommerce checkout button', async () => {
+        const style = {
+            layout: 'aaa',
+            color: 'aaa',
+            shape: 'aaa',
+            height: 5,
+            label: 'aaa',
+            tagline: true,
+        };
+
+        paypalOptions = {
+          style: style as PaypalButtonStyleOptions,
+        };
+
+        await strategy.initialize({ ...options, paypalCommerce: paypalOptions });
+
+        expect(paypal.Buttons).toHaveBeenCalledWith({
+            createOrder: expect.any(Function),
+            onApprove: expect.any(Function),
+            style: { height: 25 },
+        });
+    });
+
+    it('validate style (height, tagline) for PaypalCommerce checkout button', async () => {
```

Comment:
`it calls paypal API with height maximum of 55`

Reply:
Thank you!

Reply:
But I test here (and in the previous function) validation params in style. So in 'options' can be different style from clients who don't use typescript. Paypal's script and smart buttons don't work with wrong style, although these are just styles. So I wanted validate data and replace wrong params.
I'll create describe with validate params in different cases